packages/data/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "@wordpress/data",
+	"version": "0.0.1",
+	"description": "Data Redux module for WordPress",
+	"author": "WordPress",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"data",
+		"redux"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/data/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "src/index.js",
+	"dependencies": {
```

Comment:
Tests use `deep-freeze` and `enzyme`. Do we need to reflect that anywhere in this file as a dependency?

Reply:
Given that we are going to move those packages elsewhere one day, let's keep all dependencies listed. I will add them.

Reply:
I forgot that `enzyme` is already included in `@wordpress/scripts` by design.

Reply:
I think we should be explicit :) If we use `enzyme` directly here in our tests, it should be a direct dependency